app/models/fax.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Fax
+  delegate \
+    :sfax_api_key,
+    :sfax_encryption_key,
+    :sfax_init_vector,
+    :sfax_username,
+    to: :"Rails.application.secrets"
+
+  def self.send_fax(number:, file:)
+    new(number, file).send_fax
+  end
+
+  def initialize(number, file)
+    @number = number
+    @file = file
+  end
+
+  def send_fax
+    sfax.send_fax(number, file)
+  end
+
+  protected
```

Comment:
I'm completely cool adjusting for consistency - so I'll move them into the private space :). All good! 👍